.env.example

```diff
@@ -44,8 +49,3 @@ PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
-
-SOLDER_REPO_LOCATION=/var/www/mods.solder.test/
```

Comment:
where did these go?

Reply:
oh my bad ill re add them, seems i missed a couple things

Reply:
yeah there's a bunch of files also removed like the license

Reply:
composer.json is also messed up

Reply:
yeah ive just relised this, it sliped my mind, i just transfered over the actual app to a new laravel 8 instance

Reply:
lemme fix all this real quick

Reply:
no rush, just browsing the changed file list on Github

Reply:
is there anything else you can see that i need to add back or fix?

Reply:
thank you for going through this for me as well it helps to have a second eye to see all this

Reply:
deleted:
```
.env.testing
.travis.yml
config/solder.php
phpspec.yml
storage/github-api-cache/.gitignore 
storage/debugbar/.gitignore
```

I think you also replaced the `public/favicon.ico`

and then the formatting in all the blade files is all over the place, probably wanna add this to the editorconfig and reformat them
```yaml
[*.blade.php]
indent_size = 2
```

Reply:
actually I think they use tabs right now, and you used spaces

Reply:
ah, ill re add those file and fix my indent errors, thanks for pointing all this out to me